src/utils/get-scripts.js

```diff
@@ -5,6 +5,12 @@ import lodash from 'lodash';
 
 const scriptCache = {};
 
+export function clearCache() {
+  Object.keys(scriptCache).forEach((key) => {
```

Comment:
Why not declare `scriptCache` as `let` and directly `= {}` as `clearCache`?

Reply:
so we can maintain the same reference/object. it doesn't make a difference right now, but if for example we added a test to check the cache explicitly, we'd want to keep the same reference outside this file.